<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>422</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nome</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_nome"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Matricula</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_matricula"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_inserir">
           <property name="text">
            <string>Inserir na Tabela</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_OrdenarNome">
           <property name="text">
            <string>Ordenar por Nome</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_OrdenarMatricula">
           <property name="text">
            <string>Ordenar por Veterano</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_OrdenarCurso">
         <property name="text">
          <string>Ordenar por Curso</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tbl_dados">
       <column>
        <property name="text">
         <string>Nome</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Matrícula</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Veterano</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Curso</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>16</y>
      <width>181</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Quantidade de Alunos em:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Engenharia Elétrica</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_EngenhariaEletrica"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Administração</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_Administracao"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Computação</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_Computacao"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Automação Industrial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_AutomacaoIndustrial"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Contabilidade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_Contabilidade"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Design de Interiores</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_DesignDeInteriores"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Engenharia Civil</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_EngenhariaCivil"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Telecomunicações</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_Telecomunicacoes"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Geoprocessamento</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_Geoprocessamento"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Engenharia Química</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_EngenhariaQuimica"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSalvar">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuSalvar"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
